ansible/roles/jupyterhub/files/jupyterhub_config_lti11.py

```diff
@@ -43,34 +43,12 @@
 
 c.JupyterHub.cookie_secret_file = os.path.join(data_dir, 'jupyterhub_cookie_secret')
 
-# The instructor1 and instructor2 users have access to different shared
```

Comment:
If the load groups function is an append of an existing dictionary then does removing this configuration option have the JupyterHub create an empty dictionary by default or do we have to create one explicitly?

Reply:
Yes, this attribute has a default value because Jupyterhub is using the traitlets configuration files.
https://github.com/jupyterhub/jupyterhub/blob/master/jupyterhub/app.py#L301
https://traitlets.readthedocs.io/en/stable/config.html#python-configuration-files